admin/routes/service-manager.js

```diff
@@ -11,10 +11,12 @@ router.get('/home', isAuthenticated(rolesConfig.ROLE_SERVICE_MANAGER), (req, res
 router.get('/check-windows', isAuthenticated(rolesConfig.ROLE_SERVICE_MANAGER), (req, res, next) => serviceManagerController.getCheckWindows(req, res, next))
 router.get('/check-windows/remove/:checkWindowId', isAuthenticated(rolesConfig.ROLE_SERVICE_MANAGER), (req, res, next) => serviceManagerController.removeCheckWindow(req, res, next))
 router.get('/check-windows/add', isAuthenticated(rolesConfig.ROLE_SERVICE_MANAGER), (req, res) => serviceManagerController.getCheckWindowForm(req, res))
-router.get('/check-windows/edit/:id', isAuthenticated(rolesConfig.ROLE_SERVICE_MANAGER), (req, res, next) => serviceManagerController.getEditableCheckWindowForm(req, res, next))
+// router.get('/check-windows/edit/:id', isAuthenticated(rolesConfig.ROLE_SERVICE_MANAGER), (req, res, next) => serviceManagerController.getEditableCheckWindowForm(req, res, next))
```

Comment:
Do we need this commented line? If not, it can be removed.

Reply:
@danwilliams it's removed now